Makefile

```diff
@@ -0,0 +1,27 @@
+VERSION?=$(shell git describe --tags --dirty=-dirty)
+export BP_UNDER_TEST ?= "java-buildpack:$(VERSION)"
```

Comment:
```suggestion
VERSION?=$(shell git describe --tags  --abbrev=0 | cut -c2-  )
export BP_UNDER_TEST ?= "paketobuildpacks/java:$(VERSION)"
```

Just tried it out locally.
`git describe --tags --dirty=-dirty` returns `v14.1.0-6-gd9aec1b-dirty`
`git describe --tags  --abbrev=0 | cut -c2-` returns `14.1.0`

Reply:
with the suggested fix, I could run it locally, and it was pretty smooth; even for developers not using CI
👍